test/DynamoCoreTests/Models/DynamoModelCommandsTest.cs

```diff
@@ -0,0 +1,294 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Xml;
+using CoreNodeModels;
+using Dynamo.Graph.Nodes;
+using Dynamo.Graph.Nodes.ZeroTouch;
+using Dynamo.Models;
+using Dynamo.Selection;
+using Dynamo.Utilities;
+using NUnit.Framework;
+using static Dynamo.Models.DynamoModel;
+
+namespace Dynamo.Tests.ModelsTest
+{
+    /// <summary>
+    /// This test class will contain test method for executing methods in the DynamoModel class
+    /// </summary>
+    [TestFixture]
+    class DynamoModelCommandsTest : DynamoModelTestBase
+    {
+        /// <summary>
+        /// This test method will execute the ForceRunCancelImpl method from the DynamoModel class
+        /// </summary>
+        [Test]
+        [Category("UnitTests")]
+        public void ForceRunCancelImplTest()
+        {
+            //Arrange
+            Guid newNodeGuid = Guid.NewGuid();
+            var command = new DynamoModel.ForceRunCancelCommand(true, true);
+   
+            //Act
+            CurrentDynamoModel.ExecuteCommand(command);
+
+            //Assert
+            Assert.IsNotNull(command);
+        }
+
+        /// <summary>
+        /// This test method will execute the CreateAndConnectNodeImpl method from the DynamoModel class
+        /// </summary>
+        [Test]
+        [Category("UnitTests")]
+        public void CreateAndConnectNodeImplTest()
+        {
+            //Arrange
+            Guid newNodeGuid = Guid.NewGuid();
+            //This guid belongs to a CoreNodeModels.CreateList node inside the .dyn file
+            Guid existingNodeGuid = new Guid("81c94fd0-35a0-4680-8535-00aff41192d3");
+            double x = 100;
+            double y = 190;
+
+            string openPath = Path.Combine(TestDirectory, @"core\DetailedPreviewMargin_Test.dyn");
+            RunModel(openPath);
+
+            //This command is created with CreateAsDownstreamNode flas as true
+            var cmdOne = new DynamoModel.CreateAndConnectNodeCommand(newNodeGuid, existingNodeGuid,
+                "CoreNodeModels.CreateList", 0, 0, x, y, true, false);
+
+            //This command is created with CreateAsDownstreamNode flas as false
```

Comment:
same here

Reply:
@QilongTang 
Comment fixed in the commit: https://github.com/DynamoDS/Dynamo/pull/10657/commits/e1fa29e30eef851af322f92301c437230be5ff30